scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -224,7 +225,7 @@ class Framework extends SbtFramework {
       new AnnotatedFingerprint {
         def annotationName = "org.scalatest.WrapWith"
         def isModule = false
-      })
+      }).asInstanceOf[Array[Fingerprint | Null]]
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.